api/RTCIceTransport.json

```diff
@@ -152,6 +152,58 @@
           }
         }
       },
+      "gatheringstatechange_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCIceTransport/gatheringstatechange_event",
+          "description": "<code>gatheringstatechange</code> event",
+          "support": {
+            "chrome": {
+              "version_added": false
```

Comment:
This was [add in Chrome 70](https://storage.googleapis.com/chromium-find-releases-static/d78.html#d78d9f90c87a37772bbc55a72fb085d2c86faa85).

Reply:
Same question. Or should I just assume webview has things unless specifically told otherwise, when mobile has it?

Reply:
Yes, webview has it when mobile has it. [Here's a list of what webview doesn't have](https://cs.chromium.org/chromium/src/android_webview/tools/system_webview_shell/test/data/webexposed/not-webview-exposed.txt). This is an input file to our testing infrastructure. It shares a folder with an explicit list of what is in webview. The chance of both being wrong is extremely low. If you ever find such a case, ping me and I'll open a ticket on. A month or so ago, I submitted PRs to make sure BCD matched these.

Reply:
Thanks for that link.